.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    open-pull-requests-limit: 20
```

Comment:
Do we know how long the initial backlog is? Will 20 per month keep up with the pace of new arrivals? 

Do you know how the top 20 updates will be sorted and chosen? (date created first? major first? ( / minor first?) / alphabetically first?)

Reply:
I do not know how long the backlog is, but we can always adjust this if necessary. With the grouping of minor/patch, it's possible that 20 will be plenty.

I don't see anything in the documentation about how it decides the ordering (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#frequency-of-dependabot-pull-requests). Let me know if you find something!